### PR TITLE
Images

### PR DESCRIPTION
**Pull request by jackson14furie**
Files to Look at:
Home.js
Form.js
ForumPosts.js

### CODE REVIEW

**Review by KennaReed — CHANGES_REQUESTED** [review]
I know we talked about this during our meeting this morning, but I will just record what we talked about for record keeping. 
We can't pull this pull request into main for a couple reasons: 
- It does not pass Travis Checks 
- Does not connect to MongoDB remotely, only locally
- Choice of needed image format is unclear, so it would make it difficult for users to actually use
- No error handling to prevent users from choosing the wrong image format
Considering the time it took for this PR to get up (8 weeks), we decided there is not enough time in the remaining Sprint (5 days) to make the needed changes. Therefore, we have scrapped the feature and already have its patchwork in place.

**Review by billc2501 — CHANGES_REQUESTED** [review]
As commented earlier, these changes are not approved by Travis CI. It seems like a decent start, following proper coding styles and has some logic, but not enough to be presentable and surely not enough to be integrated in main.

File: coverage/lcov-report/base.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+  font-size: 10px;
+  color: #555;
+  background: #E8E8E8;
+  padding: 4px 5px;
+  border-radius: 3px;
+  vertical-align: middle;
+}
+
+div.path a:link, div.path a:visited { color: #333; }
```
Comment (areed08) [inline coverage/lcov-report/base.css:68]: What is all this .coverage stuff? I've never heard of it before and I'm just curious

File: src/Home.js [hunk 2]
```diff
@@ -22,9 +22,10 @@ function DisplayPost(props) {
           <p className={styles.datePosted}>
             Posted On: {formatDate(props.post.DatePosted)} </p>
         </div>
-
+	
         <p className={styles.description}> {props.post.content} </p>
         {handleEvent(props.post)}
+        <img src={props.post.image} alt={props.post.image} border= '1px' border-radius= '4px' padding= '5px' width= '150px'/>
```
Comment (areed08) [inline src/Home.js:28]: Might be nice to have an image.css file for all the sizing instead of having it on one line

**Review by areed08 — CHANGES_REQUESTED** [review]
McKenna and Billy both said it best and because of those reasons, this will not be added to main. If this would have been submitted on time there may have been time to fix these errors as this is a decent start but there is not enough time.

File: coverage/lcov-report/sorter.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+            rows[i].data = loadRowData(rows[i]);
+        }
+    }
+    // sorts the table using the data for the ith column
+    function sortByIndex(index, desc) {
+        var key = cols[index].key,
+            sorter = function(a, b) {
+                a = a.data[key];
+                b = b.data[key];
+                return a < b ? -1 : a > b ? 1 : 0;
```
Comment (dathash) [inline coverage/lcov-report/sorter.js:86]: This ternary seems pretty hard to read. Maybe some parentheses or just breaking it out into some statements.

**Review by dathash — COMMENTED** [review]
Like the others said. Good code, it just doesn't quite solve the problem and has some issues with Travis.

**Pull request closed without merging**